test/testdata/generate.sh

```diff
@@ -129,15 +129,10 @@ build_kb
 # [Deprecated] - Project version 2 uses plugin go/v2 (default).
 scaffold_test_project project-v2 --project-version=2
 
-# [Currently, default CLI plugin] - Project version 3 (default) uses plugin go/v3 (default).
-scaffold_test_project project-v3 --plugins="go/v3"
-scaffold_test_project project-v3-multigroup --plugins="go/v3"
-scaffold_test_project project-v3-declarative-v1 --plugins="go/v3,declarative"
-scaffold_test_project project-v3-config --component-config --plugins="go/v3"
-scaffold_test_project project-v3-with-deploy-image --plugins="go/v3"
-scaffold_test_project project-v3-with-grafana --plugins="go/v3"
-
-# [Next version, alpha] Project version v4-alpha
+# [Deprecated] - Project version 3 (default) uses plugin go/v3 (default).
+scaffold_test_project project-v3
+
+# [Currently, default CLI plugin] - [Next version, alpha] Project version v4-alpha
 scaffold_test_project project-v4 --plugins="go/v4"
 scaffold_test_project project-v4-multigroup --plugins="go/v4"
 scaffold_test_project project-v4-declarative-v1 --plugins="go/v4,declarative"
```

Comment:
Should we replicate the `-with-deploy-image` and `-with-grafana` samples for go/v4?

Reply:
Forgot to expand the file to see that it already does :facepalm: - ignore this